spec/acceptance/ios_ntp_config_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper_acceptance'
+
+describe 'ios_ntp_config' do
+  before(:all) do
+    skip "this device #{device_model} does not support update_calendar" if ['2960', '3650', '3750'].include?(device_model)
```

Comment:
You shouldn't be skipping he entire test here, just update_calender. I now that it's the only addition right now but here will likely be more in the future.